agent/multi_tool_agent/tools/knowledge_base.py

```
@@ -754,14 +763,25 @@ def get_event_info(query: str = "", tool_context: ToolContext = None) -> dict:
             }
         
-        # Continue with existing event_info logic...
+        elif any(word in query_lower for word in ['giveaway', 'giveaways', 'prizes']):
+            return {
+                "message": f"**Exciting Giveaways at LinkedIn Local Nigeria:**\n\n"
+                          f"🎁 **Giveaways:** {event['giveaways']}\n\n"
+                          f"**📢 How to Participate:**\n"
+                          f"  • Stay attentive during the event for announcements\n"
+                          f"  • Follow instructions provided by organizers\n"
+                          f"  • Engage actively in sessions and networking\n\n"
+                          f"*Don’t miss your chance to win laptops, merchandise, internships, or even a plot of land!*"
```

Ruff (0.13.1)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)